app/controllers/api/subcollections/cloud_tenants.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module CloudTenants
+      def cloud_tenants_query_resource(object)
+        object.respond_to?(:cloud_tenants) ? object.cloud_tenants : []
```

Comment:
Could you mix this into a collection that doesn't respond to `:cloud_tenants`? If not, does `object.cloud_tenants` suffice?